test/extended/util/db/mysql.go

```diff
@@ -80,6 +81,11 @@ func (m MySQL) QueryPrivileged(oc *util.CLI, query string) (string, error) {
 			query, masterConf.Env["MYSQL_DATABASE"])).Output()
 }
 
+// QueryPrimary queries the database on primary node as a regular user.
+func (m MySQL) QueryPrimary(oc *util.CLI, query string) (string, error) {
+	return "", errors.New("not implemented")
```

Comment:
maybe we can break this into two interfaces.... the original intent was to re-use functions that are common between databases.